src/test/java/com/marizueva/laboratory/hw5/pages/CommonPage.java

```diff
@@ -0,0 +1,36 @@
+package com.marizueva.laboratory.hw5.pages;
+
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CommonPage extends BasePage {
+    public CommonPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(css = "ul.uui-navigation.nav.navbar-nav.m-l8 a.dropdown-toggle")
+    WebElement headerItemService;
+
+    @FindBy(linkText = "DIFFERENT ELEMENTS")
+    WebElement subMenuFromHeaderDifferentElementsItem;
+
+    @FindBy(linkText = "USER TABLE")
+    WebElement subMenuFromHeaderUserTableItem;
+
+    public void clickOnServiceFromHeader() {
+        headerItemService.click();
+    }
+
+    public void goToServiceSubMenuFromHeader(String category) {
+        if (category.equals("Different Elements")) {
+            subMenuFromHeaderDifferentElementsItem.click();
+        } else if (category.equals("User Table")) {
+            subMenuFromHeaderUserTableItem.click();
+        } else {
+            throw new NotFoundException("no such element" + category);
```

Comment:
forgot **space** after **element** in the exception message

Reply:
fixed